src/sentry/incidents/logic.py

```diff
@@ -1121,6 +1135,8 @@ def get_alert_rule_trigger_action_integration_object_id(type, *args, **kwargs):
         return get_alert_rule_trigger_action_slack_channel_id(*args, **kwargs)
     elif type == AlertRuleTriggerAction.Type.MSTEAMS.value:
         return get_alert_rule_trigger_action_msteams_channel_id(*args, **kwargs)
+    elif type == AlertRuleTriggerAction.Type.PAGERDUTY.value:
```

Comment:
@ceorourke We should refactor code so that we don't call this function at all for PagerDuty